### DISCUSSION TITLE
simbot目前支持扫码登录吗？

### DISCUSSION THREAD

**Discussion by ikaros143** [simple-robot/Discussions#126] [Q&A]
### 涉及的编程语言

Java

### 涉及的组件库

mirai (QQ)

### 疑问描述

mirai扫码要2.15版本的，看simbot官网新版是2.14版本的，有办法调用扫码的方法吗？

**Comment by ForteScarlet** [comment]
mirai还未正式发布 `v2.15.0` ，simbot组件会在正式版本更新后更新，目前没有直接的扫码登录API。

但是可以自行更新mirai依赖版本后，通过 https://github.com/simple-robot/simbot-component-mirai/pull/134 来间接实现，只不过不能再使用配置文件来注册bot了，而是需要在代码里手动注册。
